worlds/alttp/test/__init__.py

```diff
@@ -1,8 +1,99 @@
 import unittest
 from argparse import Namespace
 
-from BaseClasses import MultiWorld, CollectionState
+from BaseClasses import MultiWorld, CollectionState, ItemClassification
 from worlds import AutoWorldRegister
+from ..Items import item_factory
+
+
+class TestBase(unittest.TestCase):
```

Comment:
While we're here, do we want to move this and the other base class to a new `bases.py` so it follow convention? Or is that out of scope for this PR